components/ILIAS/Authentication/classes/class.ilObjAuthSettingsGUI.php

```diff
@@ -846,23 +846,13 @@ public function executeCommand(): void
                 $this->ctrl->forwardCommand($cas_settings);
                 break;
 
-            case 'ilauthloginpageeditorgui':
+            case 'ilauthpageeditorgui':
 
                 $this->setSubTabs('authSettings');
                 $this->tabs_gui->setTabActive('authentication_settings');
                 $this->tabs_gui->setSubTabActive('auth_login_editor');
 
-                $lpe = new ilAuthLoginPageEditorGUI($this->object->getRefId());
-                $this->ctrl->forwardCommand($lpe);
-                break;
-
-            case 'ilauthlogoutpageeditorgui':
-
-                $this->setSubTabs('authSettings');
-                $this->tabs_gui->setTabActive('authentication_settings');
-                $this->tabs_gui->setSubTabActive('logout_editor');
-
-                $lpe = new ilAuthLogoutPageEditorGUI($this->object->getRefId());
+                $lpe = new ilAuthPageEditorGUI($this->object->getRefId(), AuthPageEditorContext::LOGIN);
```

Comment:
Is it correct to always pass a static`AuthPageEditorContext::LOGIN` context into the GUI class here, where we had a separation of these two contexts before?